core/robstoll-lib/atrium-core-robstoll-lib-jvm/src/test/resources/ch/tutteli/atrium/core/robstoll/lib/reporting/translating/Atrium_fr.properties

```diff
@@ -1,6 +1,6 @@
-ch.tutteli.atrium.translations.DescriptionAnyAssertion-TO_BE = est
-ch.tutteli.atrium.translations.DescriptionAnyAssertion-NOT_TO_BE = n'est pas
-ch.tutteli.atrium.translations.DescriptionAnyAssertion-IS_NOT_SAME = n'est pas la m�me instance que
+ch.tutteli.atrium.translations.DescriptionBasic-TO_BE = est
+ch.tutteli.atrium.translations.DescriptionBasic-NOT_TO_BE = n'est pas
+ch.tutteli.atrium.translations.DescriptionAnyAssertion-IS_NOT_SAME = n'est pas la m\u00EAme instance que
 ch.tutteli.atrium.specs.AssertionVerb-ASSERT = il applique que
-ch.tutteli.atrium.specs.reporting.translating.TranslationSupplierSpec$TestTranslatable-DATE_KNOWN=%tD �tait %<tA!!
-ch.tutteli.atrium.specs.reporting.translating.TranslationSupplierSpec$TestTranslatable-PLACEHOLDER=Caract�re de remplacement %s
+ch.tutteli.atrium.specs.reporting.translating.TranslationSupplierSpec$TestTranslatable-DATE_KNOWN=%tD �tait %<tA!!
```

Comment:
That does not look right, but specs are green, so I guess its OK :laughing:

Reply:
Does not look right at all!
I had encoding issues with même, which I could only solve by replacing it with an escaped unicode character. I am curios why there is no test failure for this one…

Reply:
The problem is actually intellij, it saves properties file as UTF-8 where resource-bundle properties should be saved as ISO-8859-1

Reply:
I found the problem, spek1 has the problem that it does not execute two branches with the same name. Could you please fix this and the next line and I will point out in another comment where the fix needs to apply

Reply:
Done. But please see comment below.